x-pack/plugin/build.gradle

```diff
@@ -227,6 +227,9 @@ testClusters.configureEach {
 
   requiresFeature 'es.index_mode_feature_flag_registered', Version.fromString("8.0.0")
   requiresFeature 'es.inference_rescorer_feature_flag_enabled', Version.fromString("8.10.0")
+  if (BuildParams.isSnapshotBuild() == false) {
```

Comment:
Is this if statement redundant? (the flag is always enabled in snapshot builds)

Reply:
That is true, but this is how we do it in the other tests, that's why I did this here too. I am happy to clean it here and keep it inline with the other feature flags. Any objections to remove the conditional?

Reply:
Sounds good, thanks